lib/elixir/lib/task.ex

```diff
@@ -12,46 +12,46 @@ defmodule Task do
       res  = do_some_other_work()
       res + Task.await(task)
 
-  Tasks spawned with `async` can be waited on by their caller
+  Tasks spawned with `async` can be awaited on by their caller
```

Comment:
This is a valid point. As of the this PR, the docs currently use the terms `caller`, `calling process`, `owner`, and `owning process` somewhat interchangeably. @josevalim Should something be done about it?